src/coreclr/jit/liveness.cpp

```diff
@@ -229,7 +249,7 @@ void Compiler::fgPerNodeLocalVarLiveness(GenTree* tree)
                     break;
                 }
 
-                fgMarkUseDef(tree->AsLclVarCommon());
+                fgMarkUseDef<false>(tree->AsLclVarCommon());
```

Comment:
Nit -- I might have written this as `!lowered` too, just to make it obvious all the calls in this method use the logic version.

Reply:
Good point

```suggestion
                fgMarkUseDef<!lowered>(tree->AsLclVarCommon());
```